apps/dashboard/src/@3rdweb-sdk/react/hooks/query/useQueryWithNetwork.ts

```diff
@@ -1,50 +1,12 @@
-import { networkKeys, useDashboardEVMChainId } from "@3rdweb-sdk/react";
 import {
   type MutationFunction,
-  type QueryFunction,
   type QueryKey,
   type UseMutationOptions,
-  type UseQueryOptions,
-  type UseQueryResult,
   useMutation,
-  useQuery,
   useQueryClient,
 } from "@tanstack/react-query";
 import { useCallback } from "react";
 
-export function useQueryWithNetwork<
-  TQueryFnData = unknown,
-  TError = unknown,
-  TData = TQueryFnData,
-  TQueryKey extends QueryKey = QueryKey,
->(
-  queryKey: TQueryKey,
-  queryFn: QueryFunction<TQueryFnData, TQueryKey>,
-  options?: Omit<
-    UseQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
-    "queryKey" | "queryFn"
-  >,
-): UseQueryResult<TData, TError> {
-  const activeChainId = useDashboardEVMChainId();
-
-  const mergedOptions: Omit<
-    UseQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
-    "queryKey" | "queryFn"
-  > = {
-    ...options,
-    enabled: !!(activeChainId && options?.enabled),
-  };
-
-  const combinedQueryKey = (
-    networkKeys.chain(activeChainId) as readonly unknown[]
-  ).concat(queryKey) as unknown as TQueryKey;
-
-  return useQuery({
-    ...mergedOptions,
-    queryKey: combinedQueryKey,
-    queryFn,
-  });
-}
 export function useMutationWithInvalidate<
```

Comment:
should be able to remove this, too? all it is now is a way to not have to call `queryClient.invalidateQueries` wherever we use it, might as well just do that there? this was always a bad abstraction (and that's on me)